### ISSUE TITLE
Is this package still being maintained?

### ISSUE THREAD

**Issue by rodrigorahman** [adamlofts/mysql1_dart#151]
We haven't updated this package for a long time. Is the package still being maintained?

**Comment by brasizza** [comment]
Up!

**Comment by gopelkujo** [comment]
i don't think so, if you have alternative, please kind to share with us! i'm still looking for actively maintained mysql driver for dart.

**Comment by brasizza** [comment]
> i don't think so, if you have alternative, please kind to share with us! i'm still looking for actively maintained mysql driver for dart.

try

https://pub.dev/packages/mysql_client

**Comment by gopelkujo** [comment]
> > i don't think so, if you have alternative, please kind to share with us! i'm still looking for actively maintained mysql driver for dart.
> 
> try
> 
> https://pub.dev/packages/mysql_client

thanks for your recommendation, it's really good! i use that too. but to see the last commit at the repo is 2 years ago i just worried a lilbit. so for now i fork and maintain the repo myself to update the dependencies
